apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessPreviewList.tsx

```diff
@@ -0,0 +1,557 @@
+import { FormProvider, useForm } from "react-hook-form";
+import { useState } from "react";
+import { useMutation } from "urql";
+
+import {
+  graphql,
+  ReviewRecruitmentProcessPreviewListFragment,
+} from "@gc-digital-talent/graphql";
+import {
+  commonMessages,
+  formMessages,
+  getLocale,
+  getLocalizedName,
+} from "@gc-digital-talent/i18n";
+import {
+  Button,
+  Dialog,
+  Link,
+  PreviewList,
+  PreviewMetaData,
+  Separator,
+  Well,
+} from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { toast } from "@gc-digital-talent/toast";
+import { RadioGroup } from "@gc-digital-talent/forms";
+
+import FieldDisplay from "~/components/ToggleForm/FieldDisplay";
+import talentRequestMessages from "~/messages/talentRequestMessages";
+import processMessages from "~/messages/processMessages";
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  ApplicationStatus,
+  getQualifiedRecruitmentStatusChip,
+  getSalaryRange,
+} from "~/utils/poolCandidate";
+import useRoutes from "~/hooks/useRoutes";
+
+import { RecruitmentDate } from "./MetadataDate";
+import StatusSummary from "./StatusSummary";
+
+interface FormValues {
+  isSuspended: "true" | "false";
+}
+
+const ReviewRecruitmentProcess_Mutation = graphql(/* GraphQL */ `
+  mutation Name($id: ID!, $isSuspended: Boolean!) {
+    changeApplicationSuspendedAt(id: $id, isSuspended: $isSuspended) {
+      id
+      suspendedAt
+    }
+  }
+`);
+
+export const ReviewRecruitmentProcessPreviewList_Fragment = graphql(
+  /* GraphQL */ `
+    fragment ReviewRecruitmentProcessPreviewList on PoolCandidate {
+      id
+      expiryDate
+      finalDecisionAt
+      submittedAt
+      suspendedAt
+      removedAt
+      placedAt
+      status {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      finalDecision {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      pool {
+        id
+        name {
+          en
+          fr
+        }
+        classification {
+          group
+          level
+          minSalary
+          maxSalary
+        }
+        workStream {
+          name {
+            en
+            fr
+          }
+        }
+        language {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        department {
+          name {
+            en
+            fr
+          }
+        }
+        closingDate
+        opportunityLength {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        isRemote
+        location {
+          en
+          fr
+        }
+        securityClearance {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        processNumber
+      }
+    }
+  `,
+);
+
+interface ReviewRecruitmentProcessDialogProps {
+  recruitmentProcessQuery: ReviewRecruitmentProcessPreviewListFragment;
```

Comment:
I tyhink the type should be closer to:


```
recruitmentProcessQuery: FragmentType<ReviewRecruitmentProcessPreviewList_Fragment>;
```

Reply:
Changed in [b7e4eb3](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/b7e4eb30269e46bbc3948d9cbf267c7c15a45111)

Reply:
Oh missed that! Thanks :+1:

Reply:
Fixed in [988d3bb](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/988d3bb22efa2d2da6544b633783b3c70a3f7ab5)